python/cudf/cudf/_libxx/parquet.pyx

```diff
@@ -2,42 +2,38 @@
 
 # cython: boundscheck = False
 
-from cudf._lib.cudf cimport *
-from cudf._lib.cudf import *
-from cudf._lib.utils cimport *
-from cudf._lib.utils import *
+import cudf
+import errno
+import os
+import pyarrow as pa
+
 from libc.stdlib cimport free
 from libcpp.memory cimport unique_ptr, make_unique
 from libcpp.string cimport string
 from libcpp.map cimport map
 from libcpp.vector cimport vector
 
-import errno
-import os
-import json
-import pandas as pd
-import pyarrow as pa
-
-from cudf._lib.cudf cimport (
-    size_type
-)
+from cudf._libxx.cpp.types cimport size_type
 from cudf._libxx.table cimport Table
+from cudf._libxx.cpp.table.table cimport table
 from cudf._libxx.cpp.table.table_view cimport (
     table_view
 )
-from cudf._libxx.cpp.io.parquet cimport (
-    reader as parquet_reader,
-    reader_options as parquet_reader_options,
-)
+from cudf._libxx.move cimport move
 from cudf._libxx.cpp.io.functions cimport (
     write_parquet_args,
-    write_parquet as parquet_writer
+    write_parquet as parquet_writer,
+    read_parquet_args,
+    read_parquet as parquet_reader
+)
+from cudf._libxx.io.utils cimport (
+    make_source_info
 )
 
+cimport cudf._libxx.cpp.types as cudf_types
 cimport cudf._lib.utils as lib
```

Comment:
Can get rid of this import now 😄

Reply:
This still seems to be here?

Reply:
Am I missing something? This is the _libxx.cpp.types and not _libxx.cpp.io.types ... think they are different. maybe I am seeing something wrong?

Reply:
This is `_lib` not `_libxx` where we want to get rid of this 😄